Automattic-Tracks-iOS/Event Logging/EventLoggingUploadManager.swift

```diff
@@ -34,17 +29,18 @@ class EventLoggingUploadManager {
             return
         }
 
-        guard let fileContents = fileManager.contents(atUrl: log.url) else {
+        guard fileManager.fileExistsAtURL(log.url) else {
             let error = EventLoggingFileUploadError.fileMissing
             delegate.uploadFailed(withError: error, forLog: log)
             callback(.failure(error))
             return
         }
 
-        var request = URLRequest(url: dataSource.logUploadURL)
```

Comment:
Breaking this into a method allows it to be tested in isolation (which is why its no longer inlined)